### PR TITLE
Azure Identity March Release

### PR DESCRIPTION
**Pull request by ahsonkhan**

### CODE REVIEW

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-## 1.7.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.7.0-beta.3 (2024-03-07)
```
Comment (antkmsft) [inline sdk/identity/azure-identity/CHANGELOG.md:3]: Thanks! 3/7 and not 3/5 is intentional, correct?

**Review by antkmsft — APPROVED** [review]
Approving so that you are not blocking when you need to release.
Obviously, more changes need to come in - the changes for the release themselves, and the changelog entries for them. I may review this PR later, but it is approved (CI should not allow releasing an empty changelog anyway).

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 1 above)
Comment (ahsonkhan) [reply sdk/identity/azure-identity/CHANGELOG.md:3]: For now, yes. Giving a couple days buffer. I will check back on 3/5 and adjust if needed.

**Review by antkmsft — CHANGES_REQUESTED** [review]
No March release is planned, I think this PR should be closed then.

**Pull request closed without merging**